builtin/providers/aws/resource_aws_opsworks_instance.go

```diff
@@ -565,6 +565,10 @@ func resourceAwsOpsworksInstanceRead(d *schema.ResourceData, meta interface{}) e
 	for _, v := range instance.LayerIds {
 		layerIds = append(layerIds, *v)
 	}
+	layerIds, err = sortListBasedonTFFile(layerIds, d, "layer_ids")
```

Comment:
I don't think so. If the layer ids ordering didn't match the ordering in the tffile then Terraform wanted to update layer ids to match the tf file. This makes it so that the ordering of the layer ids always matches the ordering in the tf file